resource-packs/Catppuccin UI/create_flavors.py

```diff
@@ -142,41 +138,50 @@ def main():
             f'\'import_language_files.py\' script in the \'lang\' folder.')
         return
 
+    # Get all accent colors from the catppuccin palette library
+    accent_colors = [color.name for color in PALETTE.mocha.colors if color.accent == True]
+
+    # Create a map of red2 values since they don't exist in the catppuccin palette
+    red2 = {PALETTE.mocha.name: (181, 103, 125),
```

Comment:
Some of the cancel menu sprites have had a 2nd darker color of red used ever since the beginning of the pack since it made the visibility better (That's how it was done in the original pack). An example would be this https://github.com/catppuccin/minecraft/blob/main/resource-packs/Catppuccin%20UI/template/1.20.2/assets/minecraft/textures/gui/sprites/container/beacon/cancel.png

Reply:
I'd appreciate it if we could generate the darker red from the palette red instead of seemingly hardcoding the rgb tuple.

Reply:
I did a bit of testing around and the difference in colors is pretty similar to a -26% in lightness on paint.net, trying to figure out how to do it in the script right now.